docs/provider-configuration.md

```diff
@@ -44,6 +44,7 @@ a future release. As of the "Queens" release, OpenStack no longer exposes the
 Identity v2 API.
 
 ‡ Block Storage v3 API support was added in Kubernetes 1.9.
+  *NOTE*: Cinder CSI Supports Block Storage V3 API only.
```

Comment:
Isn't it better to add this info in cinder csi plugin doc https://github.com/kubernetes/cloud-provider-openstack/blob/master/docs/using-cinder-csi-plugin.md ?

And also formatting seems wrong (it is coming in same line as L46),  I guess there should be blank line above i guess .

Reply:
we can add this in CSI docs also, but many people land on this page for provider configuration and assumes the same configuration for intree/CSI driver.

Reply:
Looks like we need to do some refactoring here,  for now this PR should be ok once formatting is fixed. Thanks

Reply:
@ramineni updated just now